docs/user-guides/retriever.md

```diff
@@ -154,6 +154,20 @@ The results will look like this, the most relevant doc is "she smiled, with pain
 You can set the `limit` parameter (default is `10`) to control the number of the most similar documents to be retrieved.
 
 
+### Memory Profile
+
+AnnLite keeps `HNSW` indexer and `cell_table` (stores columns which will be used for filtering) in memory. We estimate the memory usage of `HNSW` indexer and `cell_table` as below:
+
+- `HNSW` indexer: 1.1 * (4 bytes * `dimension` + 8 bytes * `max_connection`), where `dimension` is the dimension of the embedding vectors, and `max_connection` is the maximum number of connections in the graph. 
+For more information about the memory usage of `HNSW` indexer, you can refer to [this](https://opensearch.org/docs/latest/search-plugins/knn/knn-index/#memory-estimation)
```

Comment:
Remove this reference. what's the motivation to add it? let user know `opensearch` can also offer knn functions?